.ivy/raise-version.sh

```diff
@@ -15,5 +15,9 @@ mvn -f docker/maven/config/pom.xml --batch-mode versions:set versions:commit -Dn
 sed -i -b -E  "s:(^\s\s\s\s<version>)(.*)(<\/version>):\1${1}\3:g" $(find . -name "pom.xml")
 
 # Update artifact version in Dockerfile
+shortVer=${1:0:4} # e.g. 13.1
+echo "introducing docker image version ${shortVer}"
+
+sed -i -b -E  "s|(axonivy/axonivy-engine:)([0-9\.]+)|\1${shortVer}|g" docker/image/Dockerfile
```

Comment:
I thought it is a nice idea to already use axonivy/axonivy-engine:13.1 but there is no tag for it available.
Could we already publish the current dev as 13.1? Or is it a bad idea @alexsuter

Reply:
Ok, then I revert this change and let it use engine:12 until something else is available. 
Maybe this project should anyway better live somewhere else (docker-samples) where it would fit better with the lifecycle of its siblings.